projectile.el

```diff
@@ -3215,7 +3215,8 @@ This command will first prompt for the directory the file is in."
   (cl-mapcan
    (lambda (project)
      (when (file-exists-p project)
-       (let ((default-directory project))
+       (let ((default-directory project)
+             (projectile-cached-project-root nil))
```

Comment:
I'll think about this fix. It seems to me that we're not fixing the real problem - if such a function doesn't exist it should be trivial to add it and simply implement `projectile-current-project-files` in terms of it. Basically the implicit project-root would be become some parameter.